app/main/helpers/search_summary_manifest.yml

```diff
@@ -85,6 +85,28 @@
   filtersPreposition:
   filterRules:
   conjunction: and
+
+
+-
+  id: Supplier type
+  labelPreposition: of type
```

Comment:
Yeah Karl popped over a bit ago and said the same thing. I don't have a better suggestion off the top of my head, and I understand the pain of trying to come up with one.